tests/assets/move-projects/multiple-signers/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+# Build the Move sources.
+smove build
+# Generate script-transactions.
+# 1. init_module(Bob)
+smove create-transaction --compiled-script-path build/multiple-signers/bytecode_scripts/init_module.mv --args signer:5FHneW46xGXgs5mUiveU4sbTyGBzmstUspZC92UhjJM694ty
+# 2. rent_apartment(Alice, Alice-Stash, Bob-Stash)
+smove create-transaction --compiled-script-path build/multiple-signers/bytecode_scripts/rent_apartment.mv --args signer:5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY signer:5GNJqTPyNqANBkUVMN1LPPrxXnFouWXoe2wNSmmEoLctxiZY signer:5HpG9w8EBLe5XCrbczpwq5TSXvedjrBGCwqxK1iQ7qUsSWFc
```

Comment:
```suggestion
BOB=5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY
ALICE=5HpG9w8EBLe5XCrbczpwq5TSXvedjrBGCwqxK1iQ7qUsSWFc

smove create-transaction --compiled-script-path build/multiple-signers/bytecode_scripts/rent_apartment.mv --args signer:$BOB signer:$ALICE signer:$BOB
```

Would this work conceptually? I haven't checked whether the names are correct (and I see we have three signers, not two)

Reply:
I like it :+1: it is way more readable. I also switched the names a bit to make it more intuitive. Maybe we can later adapt this scheme to our other examples as well. Let's discuss when you are back from holiday :blush: